index.html

```diff
@@ -35,6 +35,15 @@
                     refreshSidebar()
                 }
             })
+
+            const subMenuItem = $('<li>')
+            const link = $('<a>', { href: '#', tabIndex: -1 }).click(showSidebar)
+            const outerSpan = $('<span>', { class: 'red-ui-menu-label' })
+            const innerSpan = $('<span>').html('Export Subflow')
+            innerSpan.appendTo(outerSpan)
+            outerSpan.appendTo(link)
+            link.appendTo(subMenuItem)
+            $('#menu-item-subflow-submenu').append(subMenuItem)
```

Comment:
Should be using RED.menu.addItem API

`RED.menu.addItem('parent id', { options} )`

Suggestion - NEEDS TESTING

```suggestion
            RED.menu.addItem('menu-item-subflow', {
                id: "menu-item-subflow-export",
                label: 'Export Subflow',
                onselect: function() {
                    subMenuItem()
                }
            })
```